tests/event_status/Tribe/Events/Event_Status/JSON_LDTest.php

```diff
@@ -73,6 +74,7 @@ public function it_should_have_event_status_for_canceled() {
 
 	/**
 	 * @test
+	 * @skip The meta tested is not being hydrated correctly by get_mock_event
```

Comment:
This should not be skipped anymore, right?

Reply:
Nope - still fails because the eventStatus doesn't get added to the mock object as expected

Reply:
`1) JSON_LDTest: It_should_have_event_status_for_canceled
 Test  tests/event_status/Tribe/Events/Event_Status/JSON_LDTest.php:it_should_have_event_status_for_canceled
Function modify_event did not add eventStatus.
Failed asserting that object of class "stdClass" has attribute "eventStatus".
#1  /var/www/html/wp-content/plugins/the-events-calendar/tests/event_status/Tribe/Events/Event_Status/JSON_LDTest.php:64
#2  Tribe\Events\Event_Status\JSON_LDTest->it_should_have_event_status_for_canceled
#3  {closure}`

Reply:
now fixed